checkita-core/src/main/scala/ru/raiffeisen/checkita/storage/Models.scala

```diff
@@ -49,6 +50,16 @@ object Models {
     val uniqueFieldNames: Seq[String] = Seq("jobId", "checkId", "referenceDate").map(camelToSnakeCase)
   }
 
+  sealed abstract class JobResult extends DQEntity {
```

Comment:
Looks like this abstract class is overabundant: there is only one final case class that extends this one and we can extend it directly from DQEntity.